File: ServidorPublicacion/prototipoPublicacionServidor/src/main/java/com/felipeyjuanr/servidor/model/Libro.java
package com.felipeyjuanr.servidor.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.RequiredArgsConstructor;
import java.time.LocalDateTime;


/**
 * @author Juan y Felipe R
 */
@Data
@RequiredArgsConstructor
@AllArgsConstructor
@Builder
public class Libro
{
    //Atributos
    private int cantidadPaginas;
    private double precio;
    private String titulo;
    private LocalDateTime fechaCreacion;
    private String autor;
}

File: ClientePublicacion/Cliente/Cliente/Model/Libro.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Cliente.Model
{
    public class Libro
    {
        public int CantidadPaginas { get; set; }
        public double Precio { get; set; }
        public string Titulo { get; set; }
        public DateTime FechaCreacion { get; set; }
        public string Autor { get; set; }

        public Libro(int cantidadPaginas, double precio, string titulo, DateTime fechaCreacion, string autor)
        {
            CantidadPaginas = cantidadPaginas;
            Precio = precio;
            Titulo = titulo;
            FechaCreacion = fechaCreacion;
            Autor = autor;
        }

    }
}
